src/components/restaurant/edit-restaurant.js

```diff
@@ -0,0 +1,251 @@
+import React from "react";
+import {Link} from "react-router-dom";
+import "./edit-restaurant.css";
+
+const EditRestaurant = () => {
+    return(
+      <div className="ttr-border p-3">
+          <div>
+              <Link to="/restaurant/:rid" className="btn btn-light rounded-pill fa-pull-left fw-bolder mt-2 mb-2 ms-2">
```

Comment:
After editing their profile (either clicking the X button or clicking save), I think it might be better to change the link to navigate back to the business account's profile (/profile/business/:rid), which will presumably be where they were originally, rather than navigating to /restaurant/:rid screen (result from search)

Reply:
I changed Link to path pointing back to /profile/business/:rid in edit-restaurant line9 12.